src/components/AppliedFilters.tsx

```diff
@@ -24,8 +24,8 @@ export const builtInCssClasses: Readonly<AppliedFiltersCssClasses> = {
   // Use negative margin to remove space above the filters on mobile
   appliedFiltersContainer: 'flex flex-wrap -mt-3 md:mt-0 mb-2',
   appliedFiltersLoading: 'opacity-50',
-  nlpFilter: 'border rounded-3xl px-3 py-1.5 text-sm font-medium text-neutral-dark mr-2 mb-2',
-  removableFilter: 'flex items-center border rounded-3xl px-3 py-1.5 text-sm font-medium text-neutral-dark mr-2 mb-2',
+  nlpFilter: 'border border-gray-200 rounded-3xl px-3 py-1.5 text-sm font-medium text-neutral-dark mr-2 mb-2',
```

Comment:
do we need to take care of margin differences with preflight/no preflight?

Reply:
the margins can be easily reset from the user's side if they supply custom resets similar to Tailwind preflight. the reason the border needed to be updated here is because our components were inconsistent in defining a color for the border. Tailwind has a default color for borders that we were sometimes relying on, and sometimes not. so, this update just makes that consistent so that if a user sets a custom reset for the border color, it doesn't change the border color for some of our components but not others. for margins, there is no "default", so we're always specifying the exact value in our components